rust/agents/relayer/src/msg/pending_message.rs

```diff
@@ -17,7 +17,13 @@ use super::{
     pending_operation::*,
 };
 
-const CONFIRM_DELAY_S: u64 = 60 * 10;
+const CONFIRM_DELAY: Duration = if cfg!(any(test, feature = "test-utils")) {
+    // Wait 5 seconds between retries in test mode
```

Comment:
Retries? Shouldn't it be `Wait 5 seconds to confirm txs in test mode`

Reply:
Oops